ignite/metrics/accuracy.py

```diff
@@ -14,11 +14,48 @@ class Accuracy(Metric):
     - `y_pred` must be in the following shape (batch_size, num_categories, ...) or (batch_size, ...)
```

Comment:
We need to update the docs and explain what do all new arguments.
Update info on input shapes in all cases.